lab_2/main.py

```diff
@@ -24,7 +33,19 @@ def get_language_profiles(texts_corpus: list, language_labels: list) -> dict or
     :param language_labels: a list of given language labels
     :return: a dictionary of dictionaries - language profiles
     """
-    pass
+    if not isinstance(texts_corpus, list) or not isinstance(language_labels, list):
+        return None
+    for i in texts_corpus:
+        if not i:
+            return None
+    for i in language_labels:
+        if not i:
+            return None
+
+    language_profiles = {}
+    for i in range(len(language_labels)):
+        language_profiles[language_labels[i]] = get_freq_dict(texts_corpus[i])
```

Comment:
what happens here?

Reply:
I get language label from the labels list by index. Then I get text from corpus by the same index and give it to function.
Then I create a language profiles by adding label as a key and the result of function(frequency dictionary) as a value